smoke_test/.python-version

```diff
@@ -0,0 +1 @@
+venv-3.10.3-sam
```

Comment:
I reran the command from this folder and now I get:
```
python smoke_test.py https://sam.dsde-dev.broadinstitute.org/
pyenv: version `venv-3.10.3-sam' is not installed (set by /Users/ajang/Firecloud/sam/smoke_test/.python-version)
```

I wonder if we could save a python virtualenv to the project so that anyone/everyone is using the same version of python.

Reply:
Just made a commit with several improvements for this.  With the way `pyenv virtualenv` works, I think it makes sense to _not_ commit a `.python-version` and instead just state the required version in the `README`.  It will be up to the user to make sure that they install the right python version using whatever means they want.   I also added a `requirements.txt` because that was missing and was needed.